### PR TITLE
keep Sync

### PR DESCRIPTION
**Pull request by FableFatale**

### CODE REVIEW

**Comment by FableFatale** [comment]
@zhongdongy  please review , ty

**Comment by zhongdongy** [comment]
Thanks for your contributions! But please resolve the following issues before this PR can be merged!

1. do not change `_blog.yml` contents directly, submit your translated blog index in `zh/_blog.yml` will be sufficient;
2. do not directly change the contents of original blog, copy to `zh/` directory and translate there;
3. keep the translated title consistent between `zh/_blog.yml` index entry and blog post;

File: _blog.yml
Comment (zhongdongy) [inline _blog.yml:1]: Please do not change this file

File: trufflesecurity-partnership.md
Comment (zhongdongy) [inline trufflesecurity-partnership.md:1]: Translations should be added to another file in `zh/` directory with the same name as its source.

File: zh/_blog.yml
Comment (zhongdongy) [inline zh/_blog.yml:1]: Translation index entry should appear in this file, ordered by its `date` property.

**Review by zhongdongy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**